packages/gateway/src/server.ts

```diff
@@ -1,9 +1,11 @@
 import { Server } from '@chainlink/ccip-read-server';
-import { abi as IResolverService_abi } from '@ensdomains/offchain-resolver-contracts/artifacts/contracts/OffchainResolver.sol/IResolverService.json';
-import { abi as Resolver_abi } from '@ensdomains/ens-contracts/artifacts/contracts/resolvers/Resolver.sol/Resolver.json';
 import { ethers, BytesLike } from 'ethers';
 import { hexConcat, Result } from 'ethers/lib/utils';
-
+import { ETH_COIN_TYPE } from './utils';
+const IResolverService_abi = require('@ensdomains/offchain-resolver-contracts/artifacts/contracts/OffchainResolver.sol/IResolverService.json')
```

Comment:
These should be imports.

Reply:
I don't know why but I get the following when I try to import

```
Could not find a declaration file for module '@ensdomains/offchain-resolver-contracts'. '/Users/makoto/work/ens/offchain-resolver/packages/gateway/node_modules/@ensdomains/offchain-resolver-contracts/index.js' implicitly has an 'any' type.
  Try `npm i --save-dev @types/ensdomains__offchain-resolver-contracts` if it exists or add a new declaration (.d.ts) file containing `declare module '@ensdomains/offchain-resolver-contracts';`ts(7016)
```